app/storybook/documentation/ui-kit/FAQ.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta } from '@storybook/blocks'
+import { SourceCode } from '../../.storybook/blocks/SourceCode'
+import * as CounterStories from '../../../../packages/ui-kit/src/components/Counter/Counter.stories'
+
+<Meta title="Getting started/FAQ" />
+
+# Frequently Asked Questions
+
+This section provides concise answers to common questions about our UI-kit library. Covering topics from setup to
+advanced customization, it's designed to assist you in making the most of our UI-kit. For queries not addressed here,
+our support team is always ready to help.
+
+## How can I access the DOM element?
+
+All UI-kit components rendering DOM elements forward their ref to the underlying DOM component. This allows for the
+retrieval of DOM elements by accessing the ref attached to these UI-kit components.
+
+<SourceCode of={CounterStories.SingleValueRefStory} shown />
```

Comment:
It's just a good pattern to follow for a React-based UI components library. There are numerous scenarios where it might be useful. In the case of Chart.js-based React components, I can think of the following:

- Access to a chart instance from the parent component for direct manipulations.
- Access to the canvas element for chart export (e.g., canvas.toDataURL()).
- Management of chart resize-related events.